### ISSUE TITLE
CVAR won't react

### ISSUE THREAD

**Issue by gerrard9099** [BatMen/csgo-retakes-splewis-convar-weapon-allocator#13]
sm_cvar sm_retakes_kevlar_probability_on_competitive_pistol_rounds "0"

Can't get it to work, i hav tried without sm_cvar aswell.

**Comment by BatMen** [comment]
If you type in console "rcon sm_retakes_kevlar_probability_on_competitive_pistol_rounds", it displays the current value. 

Then, be sure to set sm_retakes_weapon_mimic_competitive_pistol_rounds to 1. If you don't want to have 800$ pistol round and you don't want kevlar, you should use sm_retakes_weapon_kevlar_enabled 0 instead. The "on_competitive_pistol_round" vars are for competitive pistol round only (800$ pistol round).

**Comment by gerrard9099** [comment]
Indeed, it says "0" def "6", i want to disable kevlar/helmet only on pistol rounds.

**Comment by gerrard9099** [comment]
Still no fix.

**Comment by gerrard9099** [comment]
You available? :-) Still looking for a fix.

**Comment by BatMen** [comment]
If you have 
```
sm_retakes_weapon_mimic_competitive_pistol_rounds 1;
sm_retakes_kevlar_probability_on_competitive_pistol_rounds 0;
```

Then this is bug. If you have 
```
sm_retakes_weapon_mimic_competitive_pistol_rounds 0;
```

Then the **on_competitive_pistol_rounds** command are ignored. You only can use 
```
sm_retakes_weapon_kevlar_enabled 0;
```
